docs/recipes/custom-gatherer-puppeteer/test.sh

```diff
@@ -3,6 +3,6 @@
 # Make sure we're in this `docs/recipes/customer-gatherer-puppeteer` directory
 cd "$(dirname "$0")"
 
-node node_modules/.bin/lighthouse --config-path=custom-config.js https://www.example.com --output=json |
+node node_modules/.bin/lighthouse --legacy-navigation --config-path=custom-config.js https://www.example.com --output=json |
```

Comment:
This recipe won't work in FR because it uses `driver.wsEndpoint()` which doesn't exist on the FR driver. We could add the function, or instead we could add a `driver.getPage` so users can easily get the correct puppeteer page to use.

That will be done in a separate PR as part of our docs updates.